.github/workflows/auto-merge-dockers.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker Update Auto Merge
+on:
+  pull_request:
+    types: [opened, edited]
+  schedule:
+    - cron: '0 11 * * *'
+    - cron: '00 14 * * *'
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  auto_approve_and_merge:
+    runs-on: ubuntu-latest
+    if: startsWith(github.head_ref, 'demisto/') && github.repository == 'demisto/content' && contains(github.event.pull_request.labels.*.name, 'docker')
+    steps:
+      - name: Approve and auto-merge for docker update PRs
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          CONTENTBOT_GH_ADMIN_TOKEN: ${{ secrets.CONTENTBOT_GH_ADMIN_TOKEN }}
+        run: |
+          echo "Approving and merging"
+          gh pr review --approve "$PR_URL"
+          gh pr merge --squash "$PR_URL"
```

Comment:
What will happen if some check had failed (lint, validate code.pan.run)?

Reply:
Then it won't get merged. 
There will be an error in the action:
`X Pull request #<number>is not mergeable: the base branch policy prohibits the merge.`

Reply:
👍